### ISSUE TITLE
'NoneType' object has no attribute 'leaf_iter'

### ISSUE THREAD

**Issue by sunitj** [Ecogenomics/GTDBTk#245]
Hello!
I'm using the gtdbtk docker image `ecogenomic/gtdbtk:1.1.0` (thank you!) as described [here](https://ecogenomics.github.io/GTDBTk/running/docker.html) with the release89 db.

I have been able to use it successfully for a number of datasets. However, my most recent dataset that contains 394 bins (392 bacterial + 2 archeal), all at least 50% complete with less than 5% contamination, has been failing regardless of the size of my instance.

My command:
```
docker container run --rm \
    --workdir "$(pwd)" \
    --volume "${LOCAL_GTDB_DIR}":"${GTDB_DB_PATH}" \
    --volume "$(pwd)":"${GTDB_DATA_PATH}" \
    ${GTDB_DOCKER_IMAGE}:${GTDB_DOCKER_VERSION} \
    classify_wf \
        --cpus ${THREADS} \
        --extension ${BIN_FASTA_EXT} \
        --genome_dir ${GTDB_DATA_PATH}/${BIN_DIR} \
        --out_dir ${GTDB_DATA_PATH}/${GTDB_OUTPUT_DIR}
```

The error:
```
... 
[2020-04-23 10:05:43] INFO: Calculating RED values based on reference tree.
[2020-04-23 10:05:50] ERROR: Uncontrolled exit resulting from an unexpected error.
================================================================================
EXCEPTION: AttributeError
  MESSAGE: 'NoneType' object has no attribute 'leaf_iter'
________________________________________________________________________________
Traceback (most recent call last):
  File "/usr/local/bin/gtdbtk", line 492, in <module>
    gt_parser.parse_options(args)
  File "/usr/local/lib/python3.8/dist-packages/gtdbtk/main.py", line 666, in parse_options
    self.classify(options)
  File "/usr/local/lib/python3.8/dist-packages/gtdbtk/main.py", line 426, in classify
    classify.run(genomes,
  File "/usr/local/lib/python3.8/dist-packages/gtdbtk/classify.py", line 507, in run
    "'", '') for subnd in cur_node.leaf_iter()]
AttributeError: 'NoneType' object has no attribute 'leaf_iter'
================================================================================
```
Any idea what might be going on?  Here is the complete [GTDBtk log file](https://github.com/Ecogenomics/GTDBTk/files/4526039/gtdbtk.log)

**Comment by aaronmussig** [comment]
Hello,
Thanks for providing the output, this bug was fixed in version 1.1.1 so updating will resolve this.

**Comment by sunitj** [comment]
Great to know! I will test it now.
